### PR TITLE
Update customer-key-manage.md

### PR DESCRIPTION
**Pull request by sathana87**

### CODE REVIEW

File: microsoft-365/compliance/customer-key-manage.md [hunk 1]
```diff
@@ -404,19 +404,7 @@ To initiate the data purge path, complete these steps:
 
 ### Revoke your Customer Keys and the availability key for SharePoint Online, OneDrive for Business, and Teams files
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/customer-key-manage.md:405]:
```suggestion
### Revoke your Customer Keys and the availability key for SharePoint, OneDrive for work or school, and Teams files
```

File: microsoft-365/compliance/customer-key-manage.md [hunk 2]
```diff
@@ -404,19 +404,7 @@ To initiate the data purge path, complete these steps:
...
-
-3. Once your representative signs the legal document, return it to Microsoft (usually through an eDoc signature).
-
-   Once Microsoft receives the legal document, we run cmdlets to trigger the data purge which performs crypto deletion of the tenant key, site key, and all individual per-document keys, irrevocably breaking the key hierarchy. Once the data purge cmdlets complete, your data has been purged.
+Purging of SharePoint Online, OneDrive for Business and Teams files DEP is not supported for Customer Key. These multi-workload DEP is used to encrypt data across multiple workloads across all tenant users. Purging such DEP would result into data from across multiple workloads become inaccessible. If you decide to exit Microsoft 365 services altogether then you could pursue the path of tenant deletion per the documented process. See how to delete a tenant in Azure Active Directory.
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/customer-key-manage.md]:
```suggestion
Purging of SharePoint, OneDrive for work or school, and Teams files DEPs is not supported in Customer Key. These multi-workload DEPs are used to encrypt data across multiple workloads across all tenant users. Purging such a DEP would result in data from across multiple workloads becoming inaccessible. If you decide to exit Microsoft 365 services altogether, you could pursue the path of tenant deletion per the documented process. See how to [delete a tenant in Azure Active Directory](/azure/active-directory/enterprise-users/directory-delete-howto).  
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @sathana87, kindly commit the suggested changes. Many thanks.

**Comment by KCCross** [comment]
Looks good with the commit from Johan. Thanks!

**Pull request merged**